llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2804,27 +2875,42 @@ template <> Value *SPIRVToLLVM::transValueWithOpcode<OpAccessChain>(SPIRVValue *
       splits.push_back(std::make_pair(indices.size(), nullptr));
   }
 
+  Value *resultValue = nullptr;
+  Type *baseEltType = nullptr;
   if (splits.size() > 0) {
     Value *newBase = base;
+    Type *newBaseType = basePointeeType;
+
+    // TODO: This assert will be removed later.
```

Comment:
It would be more practical to use the same text in all the TODO's, so that it is easier to grep for all cases that need updating later (also something with "opaque" in the text would be helpful).

Reply:
done